dbt_superset_lineage/pull_dashboards.py

```diff
@@ -241,7 +241,10 @@ def get_exposures_dict(dashboards, exposures):
 
     exposures_orig = {exposure['url']: exposure for exposure in exposures}
     exposures_dict = [{
-        'name': dashboard['title'],
+        # remove non-word characters (unless it's space) and replace spaces with underscores
+        # required since dbt v1.3
+        'name': re.sub(r'[^\w\s]+', '', dashboard['title']).replace(' ', '_'),
```

Comment:
`\s` [matches multiple characters](https://docs.python.org/3/library/re.html#:~:text=0%2D9%5D.-,%5Cs,-For%20Unicode%20\(str) (`[ \t\n\r\f\v]`) and not just spaces, so maybe we should use space literally instead of it? However, I'm not sure this matters because the other characters probably can not occur in the title of a dashboard.

```suggestion
        'name': re.sub(r'[^\w ]+', '', dashboard['title']).replace(' ', '_'),
```